doc/release-notes.md

```diff
@@ -74,6 +74,13 @@ Due to a backward-incompatible change in the wallet database, wallets created
 with version 0.16.0 will be rejected by previous versions. Also, version 0.16.0
 will only create hierarchical deterministic (HD) wallets.
 
+Custom wallet directories
+---------------------
+The ability to specify a directory other than the default data directory in which to store
+wallets has been added. An existing directory can be specified using the `-walletdir=<dir>`
```

Comment:
Nit, should be `-walletsdir` (plural)?

Reply:
I prefer walletdir, no real difference though imo

Reply:
`walletsdir` seems like something I'd fat finger into `walletdir` every time, prefer to leave out the extra letter :)